lib/stat_tracker.rb

```diff
@@ -0,0 +1,22 @@
+require 'csv'
+
+class StatTracker
+  attr_reader 
+  def initialize(locations)#games, teams, game_teams)
+    # @games = games
+    # @teams = teams
+    # @game_teams = game_teams
+    @locations = locations
+  end
+
+  def self.from_csv(locations)
+    # games_data = CSV.open locations[:games], headers: true, header_converters: :symbol
```

Comment:
Thumbs up on the pseudocode here, and throughout, made it much easier to follow.